app/src/main/java/space/taran/arknavigator/mvp/model/repo/kind/DocumentKindFactory.kt

```diff
@@ -11,7 +11,7 @@ import java.nio.file.Path
 
 object DocumentKindFactory : ResourceKindFactory<ResourceKind.Document> {
     override val acceptedExtensions: Set<String> =
-        setOf("pdf", "txt", "doc", "docx", "odt", "ods", "md", "zip")
+        setOf("pdf", "txt", "doc", "docx", "odt", "ods", "md")
```

Comment:
I think `.txt` will be also excluded later when we create resource kind for "plain text files". Also, files without extensions or with un-matched extension should be checked with something like [Apache Tika](https://tika.apache.org/) if they are plain text or not.